docs/docs/tasks.md

```diff
@@ -34,9 +34,9 @@ commands:
 resources:
   gpu:
     # 24GB or more vRAM
-    memory: 24GB..
-    # Two or more GPU
-    count: 2..
+    memory: 24GB
+    # Two or more GPUs
+    count: 2
```

Comment:
How it was before was correct. 
24GB or more: `24GB..` If you write just `24GB`, it means exactly `24GB`
2 GPUs or more: `2..` If you write `2`, it means exactly 2